xbmc/cores/RetroPlayer/savestates/SavestateDatabase.cpp

```diff
@@ -107,10 +107,15 @@ bool CSavestateDatabase::RenameSavestate(const std::string& path, const std::str
   return false;
 }
 
-bool CSavestateDatabase::DeleteSavestate(const std::string& path)
+bool CSavestateDatabase::DeleteSavestate(const std::string& gamePath)
```

Comment:
Out of curiosity, what do you see as the next step, now that you've changed the signature from savestate path to game path?

Reply:
Now that you said the word signature I remembered that I didn't change the header file, is that what you meant?